README.md

```diff
@@ -1 +1,36 @@
-# HELLOMICROVERSE
\ No newline at end of file
+# Set-up-a-Hello-Microverse-project
+
+# Description the project.
+In this project, I will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know them!
+
+# Built With
+
+    html 5 , css
+    visual studio code
+    
+    
+# 👤 Author1
+
+    GitHub: @Khanetmoi
+    Twitter: @InayatVan
+    LinkedIn: Khan Bitsindou
```

Comment:
- [ ] Kindly add links to your Github, Twitter and LinkedIn. Here is an example of how you can add links in readme:

`[github](https://github.com/)`